react-delegationdashboard/src/context/state.tsx

```diff
@@ -80,7 +102,17 @@ export const initialState = () => {
     denomination: denomination,
     decimals: decimals,
     dapp: {
-      provider: new WalletProvider(sessionNetwork.walletAddress),
+      provider: getItem('ledgerLogin')
+        ? new HWProvider(
+            new ProxyProvider(
+              sessionNetwork.gatewayAddress !== undefined
+                ? sessionNetwork?.gatewayAddress
+                : 'https://gateway.elrond.com/',
```

Comment:
The url should be extracted to a constant. Also, why using the URL with the trailing `/` ? But I think the proxy provider is able to handle with it

Reply:
Done